src/sagemaker/image_uri_config/djl-neuronx.json

```diff
@@ -169,6 +176,7 @@
                 "cn-north-1": "727897471807",
                 "cn-northwest-1": "727897471807",
                 "eu-central-1": "763104351884",
+                "eu-central-2": "380420809688",
```

Comment:
are there any E2E tests covering this file? we should probably trigger those before merging.